apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -0,0 +1,70 @@
+import { Match, check } from 'meteor/check';
+import { Meteor } from 'meteor/meteor';
+import { LivechatVisitors } from '@rocket.chat/models';
+
+import { API } from '../../../../api/server';
+import { Contacts } from '../../lib/Contacts';
+
+API.v1.addRoute(
+	'omnichannel/contact',
+	{ authRequired: true },
+	{
+		async post() {
+			check(this.bodyParams, {
+				_id: Match.Maybe(String),
+				token: String,
+				name: String,
+				email: Match.Maybe(String),
+				phone: Match.Maybe(String),
+				username: Match.Maybe(String),
+				customFields: Match.Maybe(Object),
+				contactManager: Match.Maybe({
+					username: String,
+				}),
+			});
```

Comment:
This shouldn't be required now since you now have this AJV validator over [here](https://github.com/RocketChat/Rocket.Chat/blob/ab7d25c00a6dde6a42d70012ae03a48f55385905/packages/rest-typings/src/v1/omnichannel.ts#L873)

Reply:
Left them intentionally, I'm not using the AJV validators since the route has both GET/POST and both accept different paylaods. So will check later when this is possible to do with the current config.

Reply:
Makes sense :+1: